Server/Plugins/ChatLog

```diff
@@ -1 +1 @@
-Subproject commit 983d23ca37baa89f7e4dc11d71502d9c059f6376
+Subproject commit cc04da381b9aa683ee3bbd7013f9d3fd909db039
```

Comment:
Yes, all submodule changes are intentional, there were substitutions in the plugins as well.